src/parsing/binast-serialize-visitor.h

```diff
@@ -918,13 +928,14 @@ inline void BinAstSerializeVisitor::VisitThisExpression(ThisExpression* this_exp
 inline void BinAstSerializeVisitor::VisitRegExpLiteral(
     RegExpLiteral* reg_exp_literal) {
   SerializeAstNodeHeader(reg_exp_literal);
-  ToDoBinAst(reg_exp_literal);
+  SerializeRawStringReference(reg_exp_literal->raw_pattern());
+  SerializeInt32(reg_exp_literal->flags());
 }
 
 inline void BinAstSerializeVisitor::VisitSwitchStatement(
     SwitchStatement* switch_statement) {
   SerializeAstNodeHeader(switch_statement);
-  ToDoBinAst(switch_statement);
+  VisitNode(switch_statement->tag());
```

Comment:
What about all the cases for the switch statement? Might also be worth an example in test.js

Reply:
updated. briefly tried to get BreakStatement working, but handling `target` is a bit weird. putting a pin in that for now.